.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,28 +1,16 @@
-# Overview
-<!-- What this PR does, and why is needed, a useful description is expected, and relevant tickets should be mentioned -->
+### Description: 
+Summarize the changes you're submitting in a few sentences, including Jira ticket ATL-xxxx if applicable.
 
-Fixes ATL-xxxx
+### Context: 
+It creates traceability on the matter tackled by the PR. It will also help the reviewer understand what the PR is about ---- linking to related issues, bug reports, or discussions for context.
 
-## Checklist
+### Alternatives Considered (optional): 
+Link to existing ADR if any. If relevant, describe other approaches explored and why this approach was chosen. Documenting why certain paths were not taken creates a knowledge base for future reference, helping prevent others from revisiting less optimal ideas.
```

Comment:
```suggestion
Link to existing ADR, if any. If relevant, describe other approaches explored and the selected approach. Documenting why the methods were not selected will create a knowledge base for future reference, helping prevent others from revisiting less optimal ideas.
```

Reply:
Also define what ADR is

Reply:
@petevielhaber , I have explicated the acronym ADR. Please check if good enough.